cms/static/cms/js/modules/cms.sideframe.js

```diff
@@ -1,14 +1,29 @@
-//##############################################################################
-// #SIDEFRAME#
-/* global CMS */
-
+/*
```

Comment:
I'm not sure about the general header here, maybe some feedback @evildmp @mkoistinen @yakky